.github/workflows/bench-localhost.yml

```diff
@@ -0,0 +1,104 @@
+name: "Bench HTC Mock on localhost"
+
+on:
+  push:
+    branches:
+      - "ts/add-bench-workflow"
+      - "main"
+  workflow_dispatch:
+    inputs:
+      ntasks:
+        description: "Number of tasks to be created"
+        default: "100"
```

Comment:
I think the default should be more like 100k or 1M

Reply:
Eventhough it is on localhost ?